modules/file_upload/cms_file_uploader_api.e

```diff
@@ -0,0 +1,78 @@
+note
+	description: "API to manage files."
+	date: "$Date$"
+	revision: "$Revision$"
+
+class
+	CMS_FILE_UPLOADER_API
```

Comment:
as suggested weeks ago, having a dedicated api class may help here, and also for future module that may want to integrate themselves with file upload module.

in fact, at some point, I think we would rename this module as   file_module .. and include it as core module.

But this would be after your project.